### DISCUSSION TITLE
Version 1.6.0

### DISCUSSION THREAD

**Discussion by deleolajide** [igniterealtime/openfire-pade-plugin#384] [General]
# This release is for Openfire 4.7.x 
## changelog
 <ul>
     <li>Move Jitsi logging to debug</li>  
     <li>Upgrade to latest Openfire 4.7.0. Set min Opefire version to 4.7.0 and build with version 4.7.0</li>    
     <li>Fixed <a href="https://github.com/igniterealtime/openfire-pade-plugin/issues/376">Issue #376 - Add support for clustering using Hazelcast plugin</a></li> 	
     <li>Fixed <a href="https://github.com/igniterealtime/openfire-pade-plugin/issues/381">Issue #381 - Rename the package for the PushInterceptor</a></li> 
     <li>Fixed <a href="https://github.com/igniterealtime/openfire-pade-plugin/issues/383">Issue #383 - Fix NPE with Bookmark packet interceptor</a></li>  	 
 </ul>

<hr /><em>This discussion was created from the release <a href='https://github.com/igniterealtime/openfire-pade-plugin/releases/tag/v1.6.0'>Version 1.6.0</a>.</em>

**Comment by primoitt83** [comment]
Thx Dele!!

Tried here and the names seems strange...

![image](https://user-images.githubusercontent.com/18406286/150347194-b6ea4507-8f3d-4d2c-af18-4e2ace10cccb.png)

**Reply by deleolajide** [reply]
Stop your openfire server, delete the pade folder and restart openfire

**Comment by ClaudeStabile** [comment]
I will give a Try ASAP. GREAT !!!
